torch/distributed/rpc/backend_registry.py

```diff
@@ -1,18 +1,18 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 
-_RPC_BACKEND_REGISTRY = {}
+_BACKEND_REGISTRY = {}
 
 
-def _get_rpc_backend_registry():
-    return _RPC_BACKEND_REGISTRY
+def _get_backend_registry():
+    return _BACKEND_REGISTRY
 
 
-def is_rpc_backend_registered(backend_name):
-    return backend_name in _get_rpc_backend_registry()
+def is_backend_registered(backend_name):
+    return backend_name in _get_backend_registry()
 
 
-def register_rpc_backend(backend_name, init_rpc_backend_handler):
+def register_backend(backend_name, init_rpc_backend_handler):
```

Comment:
do we still need `rpc` in `init_rpc_backend_handler` arg name?

Reply:
Good point. Removed all of them.